test/dynamo/test_functions.py

```diff
@@ -1435,6 +1448,58 @@ def func():
         self.assertEqual(cnts.frame_count, 1)
         self.assertEqual(cnts.op_count, 1)
 
+    def test_triton_kernel_by_hand(self):
+        @triton.jit
+        def add_kernel(
+            in_ptr0,
+            in_ptr1,
+            out_ptr,
+            n_elements,
+            BLOCK_SIZE: tl.constexpr,
+        ):
+            pid = tl.program_id(axis=0)
+            block_start = pid * BLOCK_SIZE
+            offsets = block_start + tl.arange(0, BLOCK_SIZE)
+            mask = offsets < n_elements
+            x = tl.load(in_ptr0 + offsets, mask=mask)
+            y = tl.load(in_ptr1 + offsets, mask=mask)
+            output = x + y
+            tl.store(out_ptr + offsets, output, mask=mask)
+
+        def call_triton_add(x: torch.Tensor, y: torch.Tensor, grid_type: int):
+            output = torch.zeros_like(x)
+            n_elements = output.numel()
+
+            def grid_fn(meta):
+                return (triton.cdiv(n_elements, meta["BLOCK_SIZE"]),)
+
+            if grid_type == 0:
+                grid = (x.numel(),)
+            elif grid_type == 1:
+                grid = lambda meta: (triton.cdiv(n_elements, meta["BLOCK_SIZE"]),)
```

Comment:
@jansel Even though I did not call .bind_argument on this, n_elements seem to correctly get binded. Printing the FX graph, I can see that this triton.cdiv(n_element, meta["BLOCK_SIZE"]) results in 3.
```
call_kernel = torch__dynamo_variables_functions_call_kernel((3,), l_x_, l_y_, output, 5, BLOCK_SIZE = 2);  l_x_ = l_y_ = None
```